kong/runloop/plugins_iterator.lua

```diff
@@ -175,52 +194,50 @@ local PLUGIN_GLOBAL_KEY = build_compound_key() -- all nil
 ---
 local function lookup_cfg(combos, route_id, service_id, consumer_id)
   -- Use the build_compound_key function to create an index for the 'combos' table
-
-    local key
-    if route_id and service_id and consumer_id then
-        key = build_compound_key(route_id, service_id, consumer_id)
-        if combos[key] then
-            return combos[key]
-        end
+  local key
+  if route_id and service_id and consumer_id then
+    key = build_compound_key(route_id, service_id, consumer_id)
+    if combos[key] then
+      return combos[key]
     end
-    if route_id and consumer_id then
-        key = build_compound_key(route_id, nil, consumer_id)
-        if combos[key] then
-            return combos[key]
-        end
+  end
+  if route_id and consumer_id then
+    key = build_compound_key(route_id, nil, consumer_id)
+    if combos[key] then
+      return combos[key]
     end
-    if service_id and consumer_id then
-        key = build_compound_key(nil, service_id, consumer_id)
-        if combos[key] then
-            return combos[key]
-        end
+  end
+  if service_id and consumer_id then
+    key = build_compound_key(nil, service_id, consumer_id)
+    if combos[key] then
+      return combos[key]
     end
-    if route_id and service_id then
-        key = build_compound_key(route_id, service_id, nil)
-        if combos[key] then
-            return combos[key]
-        end
+  end
+  if route_id and service_id then
+    key = build_compound_key(route_id, service_id, nil)
+    if combos[key] then
+      return combos[key]
     end
-    if consumer_id then
-        key = build_compound_key(nil, nil, consumer_id)
-        if combos[key] then
-            return combos[key]
-        end
+  end
+  if consumer_id then
+    key = build_compound_key(nil, nil, consumer_id)
+    if combos[key] then
+      return combos[key]
     end
-    if route_id then
-        key = build_compound_key(route_id, nil, nil)
-        if combos[key] then
-            return combos[key]
-        end
+  end
+  if route_id then
+    key = build_compound_key(route_id, nil, nil)
+    if combos[key] then
+      return combos[key]
     end
-    if service_id then
-        key = build_compound_key(nil, service_id, nil)
-        if combos[key] then
-            return combos[key]
-        end
+  end
+  if service_id then
+    key = build_compound_key(nil, service_id, nil)
+    if combos[key] then
+      return combos[key]
     end
-    return combos[PLUGIN_GLOBAL_KEY]
-
```

Comment:
There seems to be no contentual change, can we revert?

Reply:
This file seems to be using mixed spacing. `4` in some cases and `2` in some. I think it is better to clean it up.